abci/types/types.proto

```diff
@@ -254,6 +254,16 @@ message Event {
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "attributes,omitempty"];
 }
 
+// TxResult contains results of executing the transaction.
+//
+// One usage is indexing transaction results.
+message TxResult {
```

Comment:
just curious, why does abci contain it's own proto types while the other packages are grouped within `/proto`?

Reply:
I just haven't migrated it to `/proto`, this is on my list.